embc-app/ClientApp/src/app/shared/components/captcha/captcha.component.ts

```diff
@@ -306,6 +311,8 @@ export class CaptchaComponent implements AfterViewInit, OnInit, OnChanges {
         this.errorFetchingImg = this.createErrorTextLine(error);
         console.log('Error esponse from fetching CAPTCHA text: %o', error);
         this.cd.detectChanges();
+        // let UI know about error
+        this.serverError.emit(error);
```

Comment:
I'm wondering if we should be able to send errors to the server for troubleshooting - generally, not specific for this

Reply:
To discuss separately

Reply:
I agree however for this we don't want an open API endpoint to post logs. This server error happens for logged out users so no header token magic can apply. It would be better to proxy through the API and the logging could get collected in the BE.